### DISCUSSION TITLE
Text indexes support for Crud Service

### DISCUSSION THREAD

**Discussion by fabio-percivaldi** [mia-platform/community#130] [Ideas]
From what I can see on the console text indexes are not supported, is it something that we can still configure in some way? if it's not possibile do you think it will be a usefull feature to add to the Console/Crud Service? 
cc: @davide-sera

**Comment by fredmaggiowski** [comment]
Hi, text indexes can still be manually configured on the database; make sure you name them with [the `preserve_` prefix](https://docs.mia-platform.eu/docs/runtime_suite/crud-service/overview_and_usage#indexes) to prevent the CRUD from removing them.

As for a native support inside Console I think it could be implemented

**Reply by fabio-percivaldi** [reply]
i didn't know about the `preserve_` prefix, that's a great feature, thanks you for the hint
